src/handlers/user-start-stop.ts

```diff
@@ -11,13 +17,22 @@ export async function userStartStop(context: Context): Promise<{ output: string
     .slice(1)
     .map((teamMate) => teamMate.split(" ")[0]);
 
-  const user = comment.user?.login ? { login: comment.user.login, id: comment.user.id } : { login: sender.login, id: sender.id };
-
   if (slashCommand === "stop") {
-    return await stop(context, issue, user, repository);
+    return await stop(context, issue, sender, repository);
   } else if (slashCommand === "start") {
     return await start(context, issue, sender, teamMates);
   }
 
-  return { output: null };
+  return { status: HttpStatusCode.NOT_MODIFIED };
+}
+
+export async function userSelfAssign(context: Context): Promise<Result> {
+  const { payload } = context;
+  const { issue } = payload;
+  const deadline = getDeadline(issue);
+
+  const users = issue.assignees.map((user) => `@${user?.login}`).join(", ");
+
+  await addCommentToIssue(context, `${users} the deadline is at ${deadline}`);
```

Comment:
won't `getDeadline` return null if price label is not present?

Reply:
Good catch, I guess I'll totally skip if no deadline is set.